api/src/services/items.ts

```diff
@@ -70,6 +90,11 @@ export class ItemsService<Item extends AnyItem = AnyItem> implements AbstractSer
 	 * Create a single new item.
 	 */
 	async createOne(data: Partial<Item>, opts?: MutationOptions): Promise<PrimaryKey> {
+		if (!opts?.mutationTracker) (opts || (opts = {})).mutationTracker = this.createMutationTracker();
```

Comment:
This one liner has a lot going on and is not the most readable i think.
```suggestion
		if (!opts) opts = {};
		if (!opts.mutationTracker) opts.mutationTracker = this.createMutationTracker();
```
Or perhaps `opts` should get defaulted to `{}` in the function argument.

Reply:
Changes pushed! 🚀